app/views/welcome/index.html.erb

```diff
@@ -6,40 +6,66 @@
 
 <div class="motivational-sentence"><%= render partial: 'shared/motivational_sentence' %></div>
 
-<div class="select-urgent-services">
-  <h4>I need urgent help now!</h4>
-  
-  <%= form_with url: search_results_path, method: :get, local: true do |form| %>
-    <%= form.label :location, 'Enter your City, State, and/or Zip Code' %><br/>
-    <div class="display-flex">
-      <%= form.text_field :location, size: "40" %> 
+<div class="display-flex">
+  <div class="select-services">
+    <h4>I need urgent help now!</h4>
```

Comment:
Should the 'I need urgent help now' be below the location button? That way, it's clear that the location is for the entire form, not just the urgent help.

Reply:
Yes, let me fix that.

Reply:
Done.